Extensions/pokes.css

```diff
@@ -64,5 +71,4 @@
 #xkit-loading_pokemon {
 	text-align: center;
 	position: relative;
-	top: 50px;
```

Comment:
I had to set it to append the pokemon list rather than replacing the panel entirely because otherwise the preferences wouldn't show up. As a result there was a 50px gap between the option to check/uncheck pokemon backgrounds and the pokemon list. Looked bad.